src/App.test.jsx

```diff
@@ -8,6 +8,23 @@ import thunkMiddleware from 'redux-thunk';
 import { Provider } from 'react-redux';
 import Api from './utils/Api';
 import App from './App.jsx';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import Api from './utils/Api';
+import thunkMiddleware from 'redux-thunk';
+import { Provider } from 'react-redux';
+import rootReducer from './reducers/index';
+
+const middleware = [
+  promiseMiddleware(),
+  thunkMiddleware.withExtraArgument({ Api}),
```

Comment:
why both Thunk and thunkmiddleware.withExtraargument?

Reply:
because: multiple developers.. @SSabu what do you think we should go with?